pkg/util/utils.go

```diff
@@ -638,3 +638,17 @@ func ValidateSysctls(strSlice []string) (map[string]string, error) {
 func DefaultContainerConfig() *config.Config {
 	return containerConfig
 }
+
+func OpenCidFile(cidfile string) (*os.File, error) {
+	if cidfile == "" {
+		return nil, nil
```

Comment:
may be need to new an error when `cidfile==""`?

Reply:
Removed check, and just let it fall through to get error at next level.